pontoon/sync/changeset.py

```diff
@@ -147,6 +147,12 @@ def update_entity_translations_from_vcs(
             )
         approved_translations = []
 
+        # Unfuzzy existing fuzzy translations to prevent duplicate fuzzies.
+        for translation in db_translations:
```

Comment:
Interesting! However, that only checks that it doesn't throw an error.

I tested myself:

```
In [1]: from bulk_update.helper import bulk_update

In [2]: from pontoon.base.models import *

In [4]: trans = Translation.objects.all()[0]

In [8]: trans.approved
Out[8]: True

In [9]: trans2 = Translation.objects.all()[0]

In [10]: trans.approved = False

In [11]: trans2.fuzzy
Out[11]: False

In [12]: trans2.fuzzy = True

In [13]: bulk_update([trans, trans2])

In [14]: trans3 = Translation.objects.all()[0]

In [15]: trans3.approved
Out[15]: True

In [16]: trans3.fuzzy
Out[16]: False
```

The changes from `trans` overrode those from `trans2`, such that `approved` was changed to True, but `fuzzy` was still False.